packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -39,24 +45,14 @@ const INSPECT = process.execArgv.some((arg) => /^--inspect(?:-brk)?(?:$|=)/.test
 const DEBUG = mode !== production && process.env.DEBUG === 'true'
 const CI = process.env.CI
 const disableHMR = process.env.HMR === 'false'
+
 const {app: appConfig} = getConfig()
 
 if ([production, development].indexOf(mode) < 0) {
     throw new Error(`Invalid mode "${mode}"`)
 }
 
-// for API convenience, add the leading slash if missing
-export const EXT_OVERRIDES_DIR =
-    typeof pkg?.ccExtensibility?.overridesDir === 'string' &&
-    !pkg?.ccExtensibility?.overridesDir?.match(/(^\/|^\\)/)
-        ? '/' + pkg?.ccExtensibility?.overridesDir?.replace(/\\/g, '/')
-        : pkg?.ccExtensibility?.overridesDir
-        ? pkg?.ccExtensibility?.overridesDir?.replace(/\\/g, '/')
-        : ''
-export const EXT_OVERRIDES_DIR_NO_SLASH = EXT_OVERRIDES_DIR?.replace(/^\//, '')
-export const EXT_EXTENDS = pkg?.ccExtensibility?.extends
-export const EXT_EXTENDS_WIN = pkg?.ccExtensibility?.extends?.replace('/', '\\')
-export const EXT_EXTENDABLE = pkg?.ccExtensibility?.extendable
```

Comment:
This is primarily removal of the previous implementation of extensibility that no longer applies. So it's being removed.